extensions/grpc/runtime/pom.xml

```diff
@@ -21,6 +21,28 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-grpc-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
@cescoffier we probably don't need this, since we depend on grpc-common, which will have this now

Reply:
done.